src/legacy/core_plugins/data/public/timefilter/timefilter.test.ts

```diff
@@ -19,46 +19,38 @@
 
 import './timefilter.test.mocks';
 
-jest.mock(
-  'ui/chrome',
-  () => ({
-    getBasePath: () => `/some/base/path`,
-    getUiSettingsClient: () => {
-      return {
-        get: (key: string) => {
-          switch (key) {
-            case 'timepicker:timeDefaults':
-              return { from: 'now-15m', to: 'now' };
-            case 'timepicker:refreshIntervalDefaults':
-              return { pause: false, value: 0 };
-            default:
-              throw new Error(`Unexpected config key: ${key}`);
-          }
-        },
-      };
-    },
-  }),
-  { virtual: true }
-);
-
-jest.mock(
-  'ui/timefilter/lib/parse_querystring',
-  () => ({
-    parseQueryString: () => {
-      return {
-        // Can not access local variable from within a mock
-        // @ts-ignore
-        forceNow: global.nowTime,
-      };
-    },
-  }),
-  { virtual: true }
-);
+jest.mock('ui/chrome', () => ({
+  getBasePath: () => `/some/base/path`,
+  getUiSettingsClient: () => {
+    return {
+      get: (key: string) => {
+        switch (key) {
+          case 'timepicker:timeDefaults':
+            return { from: 'now-15m', to: 'now' };
+          case 'timepicker:refreshIntervalDefaults':
+            return { pause: false, value: 0 };
+          default:
+            throw new Error(`Unexpected config key: ${key}`);
+        }
+      },
+    };
+  },
+}));
```

Comment:
I wonder if the mock implementation is needed here anymore, vs just using `jest.mock('ui/chrome')` and letting it pickup the mocks provided in `ui/public/chrome/__mocks__`?

Reply:
Timefilter still uses `chrome` to get the default values.
And this mock is required to provide them.
I'll remove it once `timefilter` uses `npSetup.core.uiSettings`.